### DISCUSSION TITLE
Welcome to LIPs Discussions!

### DISCUSSION THREAD

**Discussion by frozeman** [lukso-network/LIPs#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect dicusss ideas about ERC725 and LIPS in general, without creating issues.
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by potions1** [comment]
Profile Rentals : CONECPT 

User A Requests Posting:

User A initiates a request to post content on User B's profile and includes a fee for this service.
Incoming Token Received:

User A sends a token representing the payment for the posting service to User B.
Token Validation:

User B validates the incoming post request to ensure it is genuine and has the appropriate to audience.
Review Content:

User B reviews the content that User A wants to post on their profile. This may include an advertisement or other types of content.
Decision by User B:

User B has the option to either accept or decline the request based on the content's appropriateness or other criteria.

If accepted, User B provides permission for the content to be posted on their profile.

If declined, the transaction is canceled, and User A's content is not posted.

Posting on Lukso Blockchain:

If User B accepts the request, the Lukso blockchain is utilized to execute the posting transaction securely.
Content on User B's Profile:

The content written by User A is now posted on User B's profile, visible to their audience.

Wrtiten by @kingzoot27

**Reply by 0xAzures** [reply]
I like it ❤️. 

I would add the possibility (maybe not the necessity) to include a fee as a reward to review the application and the invest time into thinking it over. 
On top, B can set a base fee and no applications will be accepted for review by B if base fee < minimum fee.
